Apps/phcanary/canary.json

```diff
@@ -0,0 +1,738 @@
+{
+    "appid": "00b3dc3a-ea6e-47a8-82a5-036f05d83863",
+    "name": "Canary",
+    "description": "App to Integrate with ThinkST Canary API Service",
```

Comment:
Needs a better description. Please refer to existing apps

Reply:
Fixed.